src/algebraic_topology/simplex_category.lean

```diff
@@ -5,10 +5,10 @@ Authors: Johan Commelin, Scott Morrison, Adam Topaz
 -/
 
 import category_theory.skeletal
-import tactic.linarith
 import data.fintype.sort
 import order.category.NonemptyFinLinOrd
 import category_theory.functor.reflects_isomorphisms
+import tactic.linarith
```

Comment:
Is this intended?

Reply:
No. I had to patch up lots of tactic imports by hand. I'll fix this one now.